components/function-controller/main.go

```diff
@@ -83,6 +104,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	resourceWatcherServices := resource_watcher.NewResourceWatcherServices(coreClient, envConfig.ResourceWatcherConfig)
```

Comment:
Please update README.md because still there is information about the only namespace that allows running functions (`serverless`).

Reply:
I removed it in another integration PR :)